### DISCUSSION TITLE
Is there an example/walkthrough on how to assign a Policy to resource groups that are tagged as production.

### DISCUSSION THREAD

**Discussion by ashokkumarraya** [gettek/terraform-azurerm-policy-as-code#107] [Q&A]
Hi gettek,

we have to assign the policies to particular resource groups that are tagged as production, can you please guide me on how to do this?

**Comment by ashokkumarraya** [comment]
Hi gettek, 

Do you have any solution here?

**Comment by gettek** [comment]
@ashokkumarraya, apologies for the late reply...

To simplify assignment try adding tagging conditions to the Policy rule as in the example below. This will allow you to define and assign the policy at a high level e.g. Management Group.

```json
"policyRule": {
      "if": {
        "allOf": [
          {
            "field": "type",
            "equals": "Microsoft.Resources/subscriptions/resourceGroups"
          },
          {
            "field": "[concat('tags[', parameters('tagName'), ']')]",
            "equals": "Production"
          }
        ]
      },
      "then": {
        "effect": "[parameters('effect')]"
      }
    }
```

**Reply by rayaashokkumar321** [reply]
Thanks @gettek

**Reply by gettek** [reply]
If this worked, please mark it as the answer
